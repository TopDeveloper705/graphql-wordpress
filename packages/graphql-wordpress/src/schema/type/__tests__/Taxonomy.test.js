import TaxonomyType from 'type/Taxonomy';

// eslint-disable-next-line no-underscore-dangle
const fields = TaxonomyType._typeConfig.fields;

describe('Test Taxonomy type', () => {
  test('Test name', () => {
    expect(TaxonomyType.name).toMatchSnapshot();
  });

  test('Test fields', () => {
    expect(Object.keys(TaxonomyType.getFields())).toMatchSnapshot();
  });

  test('Test resolve name', () => {
    const name = fields.name.resolve({ slug: 'category' });
    expect(name).toMatchSnapshot();
  });
});
