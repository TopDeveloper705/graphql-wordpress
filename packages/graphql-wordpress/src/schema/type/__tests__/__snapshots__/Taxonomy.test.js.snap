// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Taxonomy type Test fields 1`] = `
Array [
  "id",
  "name",
  "description",
  "types",
  "hierarchical",
  "rewrite",
  "labels",
]
`;

exports[`Test Taxonomy type Test name 1`] = `"Taxonomy"`;

exports[`Test Taxonomy type Test resolve name 1`] = `"category"`;
